### ISSUE TITLE
React & Web component - IBM logo is getting truncated at 200% as well as 400% zoom.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#7201]
### Description

Issue: 
Top links in the component are getting truncated at 200% as well as 400% zoom.

Expected:
Top links should be visible clearly on 200% and 400% zoom.

Note: This is observed in 'Short', 'Short with adjunct legal links', 'Short language only', 'Micro' and 'Micro Language only'.

### Component(s) impacted

![image](https://user-images.githubusercontent.com/60002341/134120094-7f278e83-6436-4ba3-a7bb-6f774fe341ff.png)

### Browser

Firefox

### Carbon for IBM.com version

1.25

### Business priority

High Priority = pressing release

### Application/website

Carbon for IBM

### Package

@carbon/ibmdotcom-react, @carbon/ibmdotcom-web-components

### CodeSandbox example

ex

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
